consensus/hotstuff/votecollector/combined_vote_processor_v3_test.go

```diff
@@ -752,3 +759,113 @@ func TestCombinedVoteProcessorV3_PropertyCreatingQCLiveness(testifyT *testing.T)
 		}
 	})
 }
+
+// TestCombinedVoteProcessorV2_BuildVerifyQC tests a complete path from creating votes to collecting votes and then
+// building & verifying QC.
+// We start with leader proposing a block, then new leader collects votes and builds a QC.
+// Need to verify that QC that was produced is valid and can be embedded in new proposal.
+func TestCombinedVoteProcessorV3_BuildVerifyQC(t *testing.T) {
+	t.Skip("This test is skipped due to missing RandomBeaconReconstructor impl")
```

Comment:
This test is majorly updated in https://github.com/onflow/flow-go/pull/1602, it's more of `StakingVoteProcessor` copy cat in this PR.